webviewer/src/components/WelcomeWindow/WelcomeWindow.js

```diff
@@ -16,9 +16,12 @@ export const WelcomeWindow = ({ onClose }) => {
   const handleClick = e => setIsChecked(e.target.checked);
   const handleClose = () => onClose(isChecked);
 
+  const domain = window.location.hostname;
+  const title = domain === 'localhost' ? 'Agrieos' : 'SoilMate';
```

Comment:
Please collect all such things which decide upon domain name what to show in one place. Thus we won't need to look for 'Agrieos' and 'agrieos.in' in many files

Reply:
fixed